fighthealthinsurance/ml/ml_models.py

```
@@ -2271,4 +2275,76 @@ def models(cls) -> List[ModelDescription]:
+    Backend that auto-discovers model servers via Tailscale DNS.
+
+    Looks for hosts named 'azure-{model-name}' in the Tailscale network
+    and adds them as available backends.
+    """
+
+    # Models we try to discover via Tailscale DNS
+    DISCOVERABLE_MODELS = [
+        ("fhi-legacy", "TotallyLegitCo/fighthealthinsurance_model_v0.5"),
+        ("fhi-new", "/models/fhi-2025-may-0.3-float16-q8-vllm-compressed"),
+        ("llama-scout", "meta-llama/Llama-4-Scout-17B-16E-Instruct"),
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)